programs/perpetuals/tests/anchor/cryptography.ts

```diff
@@ -0,0 +1,22 @@
+import * as sha3 from "sha3";
+import { randomBytes } from "crypto";
+import * as secp256k1 from "secp256k1";
+
+export function generateSecp256k1Keypair() {
```

Comment:
why do you need to generate a keypair yourself?

there is a Keypair object in the web3.js library, unless you have a reason for making your own?

Reply:
hahah yes good call;  we had initially set the signing scheme to be secp256k1 and I didn't see support for that in web3.js -- but we have since changed the scheme to ed25519!   removed secp256k1 utils